apps/aecore/test/aec_mining_tests.erl

```diff
@@ -156,13 +156,15 @@ setup(PoWMod) ->
     meck:new(aeu_time, [passthrough]),
     meck:expect(aeu_time, now_in_msecs, 0, 1510253222889),
     {ok, _} = aec_tx_pool:start_link(),
+    SignedTx = #signed_tx{data = #coinbase_tx{account = <<"pubkey">>},
+                          signatures = [<<"sig1">>]},
     Trees = #trees{accounts = [#account{pubkey = <<"pubkey">>}]},
     meck:expect(aec_trees, all_trees_hash, 1, <<>>),
-    meck:expect(aec_tx, apply_signed, 3, {ok, Trees}),
+    meck:expect(aec_tx, filter_out_invalid_signatures, fun(X) -> X end),
```

Comment:
Too bad we cannot remove these mecks here and use `aec_test_util:aec_keys_setup/0`... /cc @hanssv